packages/layerzero-v2/evm/oapp/test/TestHelper.sol

```diff
@@ -64,121 +64,127 @@ contract TestHelper is Test, OptionsHelper {
         executorValueCap = _valueCap;
     }
 
+    struct EndpointSetup {
```

Comment:
worth to add natspec comment?
/**
 * @dev Structure to manage and store endpoint-related configurations.
 * Used for setting up multiple endpoints in testing and deployment.
 *
 * @param endpointList Array of deployed endpoint contracts.
 * @param eidList List of endpoint identifiers (EIDs).
 * @param sendLibs Addresses of sending libraries for each endpoint.
 * @param receiveLibs Addresses of receiving libraries for each endpoint.
 * @param signers List of signer addresses for verification.
 * @param priceFeed Reference to the deployed price feed contract.
 */